Tests/Packet++Test/Tests/EthAndArpTests.cpp

```diff
@@ -7,8 +7,38 @@
 #include "ArpLayer.h"
 #include "PayloadLayer.h"
 #include "Packet.h"
+#include "OUILookup.h"
 #include "SystemUtils.h"
 
+PTF_TEST_CASE(OUILookup)
+{
+	pcpp::OUILookup lookupEngineJson;
+	PTF_ASSERT_GREATER_THAN(lookupEngineJson.initOUIDatabaseFromJson("../../3rdParty/OUIDataset/PCPP_OUIDataset.json"), 0);
+
+	PTF_ASSERT_EQUAL(lookupEngineJson.getVendorName(pcpp::MacAddress("aa:aa:aa:aa:aa:aa")), "Unknown");
```

Comment:
`pcpp::MacAddress` doesn't have an explicit c'tor, so we can also do:
```cpp
PTF_ASSERT_EQUAL(lookupEngineJson.getVendorName("aa:aa:aa:aa:aa:aa"), "Unknown");
```

Same for other assertions in this file